source/developers/how-tos/use-python-coverage-comment.rst

```diff
@@ -0,0 +1,38 @@
+How to Set Up Github Action to Monitor Python Coverage
+######################################################
+
+Most of our public repos use `CodeCov`_ to track our coverage, which is a separate service
```

Comment:
Formatting nits:

- We probably want sentence case rather than title case here. (I don't think this is in the style guide, but I usually see sentence case for these headings, and I personally think they're easier to read that way.)
- The style guide says to use long lines rather than hard wraps in our rST docs. (I really prefer hard wraps for ease of diff review, but oh well.)

Reply:
I was looking at the other How-Tos in this directory, and they use both title case and hard wraps, so I don't know what the actual recommendation is, here. 🙃

Reply:
I'd go with OEP-19 on the line wrapping: https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0019-bp-developer-documentation.html#best-practices

The title case/sentence case thing is mostly a personal preference and an unscientific vague recollection of what I'd seen elsewhere, so definitely go for consistency with the existing files on that matter. :-)